content/documentation/opencensus.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "Running Cluster with OpenCensus Tracing and Metrics"
```

Comment:
I don't see this linked from anywhere. Should it be part of the "developer documentation" ? Or another step in the normal user documentation?

Reply:
Woops, thanks

Reply:
fixed